docs/docs/how-to/sourcing-data/headless-cms.md

```diff
@@ -6,52 +6,54 @@ A _headless CMS_ is content management software that enables writers to produce
-| [Cockpit](https://getcockpit.com/)            |                                                                                  | [docs](/plugins/gatsby-plugin-cockpit)              |                                                                      |
-| [CraftCMS](https://craftcms.com/)             |                                                                                  | [docs](/plugins/gatsby-source-craftcms)             |                                                                      |
-| [Agility CMS](https://agilitycms.com/)        | [guide](/docs/sourcing-from-agilitycms/)                                         | [docs](/plugins/@agility/gatsby-source-agilitycms/) | [starter](/starters/agility/agility-gatsby-starter/)                 |
-| [Forestry](https://forestry.io/)              | [guide](/docs/sourcing-from-forestry/)                                           |                                                     |                                                                      |
-| [Gentics Mesh](https://getmesh.io)            | [guide](/docs/sourcing-from-gentics-mesh)                                        |                                                     |                                                                      |
-| [Seams-CMS](https://seams-cms.com/)           | [guide](/docs/sourcing-from-seams-cms)                                           |                                                     |                                                                      |
-| [Builder.io](https://www.builder.io/)         | [guide](/docs/sourcing-from-builder-io/)                                         | [docs](/plugins/@builder.io/gatsby/)                | [starter](https://github.com/BuilderIO/gatsby-starter-builder)       |
-| [Flotiq](https://flotiq.com/)                 | [guide](/docs/sourcing-from-flotiq/)                                             | [docs](/plugins/gatsby-source-flotiq)               | [starter](https://github.com/flotiq/gatsby-starter-blog)             |
-| [Tina](https://tina.io)                       | [guide](https://tina.io/guides/#gatsby)                                          | [docs](https://tina.io/docs/integrations/gatsby/)   |                                                                      |
-
-## How to add new guides to this section
-
-If you don’t see your preferred CMS in this list, you can [write a new guide yourself](/contributing/how-to-contribute/) or [open an issue to request it](https://github.com/gatsbyjs/gatsby/issues/new/choose).
-
-You can also [write your own source plugin](/docs/how-to/plugins-and-themes/creating-a-source-plugin/) to integrate Gatsby with a CMS that is not in the list.
+## Main CMSs
```

Comment:
"Main" isn't maybe the word we want? Maybe just "Popular CMSs"? Call it what it is 🤷